fl4health/feature_alignment/constants.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, Union
+
+from flwr.common.typing import Scalar
+from sklearn.feature_extraction.text import CountVectorizer, HashingVectorizer, TfidfTransformer, TfidfVectorizer
+
+TextFeatureTransformer = Union[CountVectorizer, TfidfTransformer, TfidfVectorizer, HashingVectorizer]
+
+# Constants that specify the data type of tabular data columns.
```

Comment:
So rather than having these be string constants and feature types be a list, I think we want to make this an enum with possible values as below. There are ways to iterate through the possible values of an enum,  so we wouldn't need the feature_types list anymore. That way we don't have to keep these in sync if we ever update anything and the types are strongly typed by the enum.